comparison_to_ros.md

```diff
@@ -0,0 +1,11 @@
+# Comparison to ROS
```

Comment:
please create a doc folder and move this there, I am sure we will have more document file in the future

Reply:
done